dags.yaml

```diff
@@ -1066,6 +1066,30 @@ bqetl_adjust:
     - impact/tier_2
     - repo/bigquery-etl
 
+
+bqetl_adjust_derived:
```

Comment:
There seems to be an `bqetl_adjust` DAG already. Could that be reused to schedule the newly added query? (it seems to have the same schedule interval and tier)

Reply:
I'll combine the two

Reply:
The other one isn't in use anymore, should I just replace the code?

Reply:
I suggest removing `scheduling` from this query: https://github.com/mozilla/bigquery-etl/blob/e719266146fb980fdab1f7a2690a64384b523a4c/sql/moz-fx-data-marketing-prod/adjust_derived/firefox_mobile_installs_v1/metadata.yaml#L8C1-L10

Reply:
And then keep `bqetl_adjust` in favour of `bqetl_adjust_derived`